### PR TITLE
feat: add module containing centralized exchanges info

### PR DESCRIPTION
**Pull request by dlipicar**
Add util for fetching list of centralized exchanges and their known addresses from [CoinCarp](https://www.coincarp.com/). The list can be updated and amended in the future. The util generates a go module for accessing that list from status-go.

This is part of https://github.com/status-im/status-desktop/issues/10251

### CODE REVIEW

**Comment by ghost** [comment]
### Pull Request Checklist
- [ ] Have you updated the documentation, if impacted (e.g. [docs.status.im](https://status.im/docs/))?
- [ ] Have you tested changes with mobile?
- [x] Have you tested changes with desktop?

File: exchanges/cmd/exchanges.txt [hunk 1]
```diff
@@ -0,0 +1,89 @@
+package exchanges
```
Comment (alaibe) [inline exchanges/cmd/exchanges.txt:1]: Is that normal? to have a txt files?

File: exchanges/exchanges.json [hunk 2]
```diff
@@ -0,0 +1,1227 @@
+[
```
Comment (alaibe) [inline exchanges/exchanges.json:1]: How did u get this list?

File: exchanges/cmd/exchanges.txt (hunk 1 above)
Comment (dlipicar) [reply exchanges/cmd/exchanges.txt:1]: I've seen a couple modules (for example, the one we use for the iso4217 fiat currency list: https://github.com/ladydascalie/currency) follow this approach for converting data downloaded from some external source to golang code. This file is the "template" used for the automatic conversion (https://pkg.go.dev/text/template)

File: exchanges/exchanges.json (hunk 2 above)
Comment (dlipicar) [reply exchanges/exchanges.json:1]: Auto-generated by the util inside `cmd`

**Review by alaibe — APPROVED** [review]

**Pull request merged**